quarkchain/proto/grpc.proto

```diff
@@ -1,27 +1,32 @@
 // SPDX-License-Identifier: Apache-2.0
-
 syntax = "proto3";
-
 package cluster;
 
-message SetRootChainConfirmedBlockRequest {
+message RootBlockHeader {
```

Comment:
you probably should just copy the proto file without any modification

Reply:
Done.